lib/interface.rb

```diff
@@ -0,0 +1,96 @@
+class Interface
+
+  attr_reader :menu_array, :order
+
+  def initialize(filepath)
+    @menu = Menu.new(filepath)
+    @order = Order.new
+    @order_array = []
+    @menu.build_menu_display
+  end
+
+  def user_interaction
+    welcome
+    build_order(@menu, @order)
+    show_order
+    order_confirm
+  end
+
+  def welcome
+    file_reader('/Users/malinnaleach/Programs/takeaway-challenge/lib/welcome.txt')
+    display(@menu.menu_array)
+  end
+
+  def file_reader(filepath)
+    File.open(filepath) {|f| puts f.read}
```

Comment:
Have a look at doing this without a block, to make this line more succinct.  Also, try to make function names verbs.